src/style.css

```
@@ -0,0 +1,119 @@
+* {
+    margin: 0;
+    box-sizing: border-box;
+    font-family: 'Roboto', sans-serif;
+}
+
+.product-card {
+    display: flex;
+    flex-direction: column;
+    justify-items: center;
+
+    background-color: white;
+    border: 1px solid #F3F3F3;
+    border-radius: 5px;
+    width: 200px;
+    height: 400px;
+}
+
+.image {
+    background-image: url(images/imac.jpeg);
+    background-repeat: no-repeat;
+    background-position: center;
+    background-size: contain;
+    width: 160px;
+    height: 134px;
+    margin: 32px 19px 0;
+}
+
+.product-card__description {
+    color:#060B35;
+    font-weight: 500;
+    font-size: 12px;
+    margin: 40px 16px 0;
+
+}
+
+.product-card__item-code {
+    margin-top: 10px;
+    margin-left: 16px;
+    font-size: 10px;
+    font-weight: 400;
+    color: #616070;
+}
+
+.product-card__mark {
+    display: flex;
+    margin-top: 16px;
+    margin-left: 16px;
+
+}
+
+.product-card__star {
+    background-image: url(images/star-active.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+    margin-right: 2px;
+    width: 16px;
+    height: 16px;
+}
+
+.product-card__mark > :nth-child(5) {
+    background-image: url(images/star.svg);
+    margin-right: 0;
+}
+
+.product-card__mark > :last-child {
+    flex-grow: 1;
+    color:#060B35;
+    font-size: 10px;
+    font-weight: 400;
+    margin-left: 25px;
+    margin-right: 16px;
+    align-self: center;
+}
+
+.product-card__price {
+    display: flex;
+    justify-content: space-between;
+    margin: 24px 16px 0;
+}
+
+.product-card__text-price {
+    color:#616070;
+    font-weight: 400;
+    font-size: 12px;
+    align-self: center;
+}
+
+.product-card__price-numbers {
+    color:#060B35;
+    font-weight: 700;
+    font-size: 16px;
+}
+
+.product-card__container__buy {
+    height: 40px;
+    line-height: 40px;
+    margin: 16px 16px;
+}
+
+.product-card__button-buy {
+    display: block;
+    text-align: center;
+    height: 100%;
+    color: white;
+    background-color:#00ACDC;
+    border: 1px solid transparent;
+    border-radius: 5px;
+    text-decoration: none;
+    font-size: 14px;
+    font-weight: 700 bold;
+}
+
+.product-card__button-buy:hover {
+    background-color: white;
+    color: #00ACDC;
+    border: 1px solid #00ACDC;
+
+}
```

```suggestion
    border: 1px solid #00ACDC;
}

```
Має бути один пустий рядок в кінці коду